app.js

```diff
@@ -0,0 +1,6279 @@
+function drawing(size) {
+    let field = document.querySelector('canvas'),
+        ctx = field.getContext('2d'),
+        width = square[0].length,
+        height = square.length,
+        scale = Math.round(512 / size);
```

Comment:
Magic number

Reply:
And in this case please use switch case structure instead

Reply:
not fixed(magic number still exist)